src/locales/es.json

```diff
@@ -21,7 +21,7 @@
     "receive": "Recibir",
     "send": "Enviar",
     "tooltip_ethereum": "Este es tu balance de MANA en Ethereum",
-    "tooltip_matic": "Este es tu balance de MANA en Polygon (antes llamado Matic)"
+    "tooltip_matic": "Este es tu balance de MANA en Polygon"
```

Comment:
Do you know why is the text different in the EN version?

Reply:
I think we used to have it like that there, then we removed it but we did it only in the english translations (probably we forgot to update the others)